.github/workflows/tests.yml

```
@@ -122,7 +122,8 @@ jobs:
           python -m pip install --disable-pip-version-check --upgrade pip
           pip install --upgrade wheel setuptools
           if [ -z "${{ matrix.numpy }}" ]; then
-            pip install --upgrade --pre Cython numpy pytest pytest-cov coveralls;
+            pip install --upgrade --pre Cython pytest pytest-cov coveralls;
+            pip install --upgrade --pre --only-binary :all: -i https://pypi.anaconda.org/scipy-wheels-nightly/simple numpy
```

```suggestion
            pip install --upgrade --pre --only-binary :all: -i https://pypi.anaconda.org/scientific-python-nightly-wheels/simple numpy
```

Scipy channel is no longer in use, the latest numpy there is from 2023-06-16.

---

I copied it from their docs ... but the page I landed on from googling is the one from 1.24 ;)